landscaper/cmd/landscaper-gardenlet/app/landscaper-gardenlet.go

```diff
@@ -0,0 +1,150 @@
+// Copyright (c) 2021 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package app
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	gardenlet "github.com/gardener/gardener/landscaper/gardenlet/pkg"
+	"github.com/gardener/gardener/landscaper/gardenlet/pkg/apis/imports"
+	importsv1alpha1 "github.com/gardener/gardener/landscaper/gardenlet/pkg/apis/imports/v1alpha1"
+	importvalidation "github.com/gardener/gardener/landscaper/gardenlet/pkg/apis/imports/validation"
+
+	gardenletconfig "github.com/gardener/gardener/pkg/gardenlet/apis/config"
+	gardenletconfigv1alpha1 "github.com/gardener/gardener/pkg/gardenlet/apis/config/v1alpha1"
+	"github.com/gardener/gardener/pkg/version/verflag"
```

Comment:
This pkg is removed with https://github.com/gardener/gardener/pull/3626.

```suggestion
	"k8s.io/component-base/version/verflag"
```